packages/core/stories/elements/Link/Link.stories.mdx

```diff
@@ -14,27 +14,28 @@ import * as stories from './Link.stories.js';
 />
 
 # Links
-Links are used to embed actions or pathways to more information in a sentence or on a page.
 
-## Variations
+Links are used to embed actions or pathways to more information in a sentence or on a page.
 
-#### Inline Link
-**Usage:** Use when you are linking just a few words of text or when you are doing inline links.
+### Inline Link
 
-**Style:** [To-do: Regular inline links should also have underline decoration.. Look into generating the appropriate CSS]
+- **Usage:** Use when you are linking just a few words of text or when you are doing inline links.
+- **Style:** Inline links take the style of the surrounding text, adding only the blue color and an underline.
 
-<Canvas>
+<Canvas className="borderless">
   <Story
     name="link">
-      <a href="#">Inline Link</a>
+      <p>Some content and then an <a href="#">inline link</a>, followed by more content.</p>
   </Story>
   <Story story={stories.linkCSS} />
 </Canvas>
 
-#### Decorative Link
-**Usage:** Used for standalone links that require more emphasis.
+---
 
-**Style:**
+### Decorative Link
+
+- **Usage:** Used for standalone links that require more emphasis, usually inside `ma__section-links`.
+- **Style:** the decorative link is slightly larger in size and appended with an arrow icon.
```

Comment:
Can we also add the hover state to "Style" for decorative link

Reply:
Sure